flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/PulsarSourceBuilder.java

```diff
@@ -311,6 +311,11 @@
      * return {@link Boundedness#CONTINUOUS_UNBOUNDED} even though it will stop at the stopping
```

Comment:
For consistency purposes, I think it would be great if the text from line 89-93 and and 303-307 can be copied over, except for the different methods of course.

Reply:
Not sure if I understood correctly, I copied line 89-93 to replace line 303-307 except for this sentence " For
example the following PulsarSource stops after it consumes up to a event time when the Flink
 started."   XD